Bottom: 74e6eadeb7ddbcd6f0e66118f8748396b5f55b89
Top:    3174dfced34e2f2e6b8e33b5a55557a1a99a27a3
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-07 15:20:03 -0500

Add new host buildbot-worker.novatech-llc.com

Runs buildbot worker containers


---
diff --git a/ansible-playbook/host_vars/builder.novatech-llc.com/buildbot.yml b/ansible-playbook/host_vars/builder.novatech-llc.com/buildbot.yml
new file mode 100644
index 00000000..9a8e64a5
--- /dev/null
+++ b/ansible-playbook/host_vars/builder.novatech-llc.com/buildbot.yml
@@ -0,0 +1 @@
+buildbot_uid: 1001
diff --git a/ansible-playbook/host_vars/builder.novatech-llc.com/docker.yml b/ansible-playbook/host_vars/builder.novatech-llc.com/docker.yml
new file mode 100644
index 00000000..3c45191d
--- /dev/null
+++ b/ansible-playbook/host_vars/builder.novatech-llc.com/docker.yml
@@ -0,0 +1 @@
+worker_backend_network: backend
diff --git a/ansible-playbook/production b/ansible-playbook/production
index bae6e4f9..9991708e 100644
--- a/ansible-playbook/production
+++ b/ansible-playbook/production
@@ -1,3 +1,5 @@
+builder.novatech-llc.com ansible_host=172.16.64.4 ansible_user=ansibleremote
+
 [bacula]
 daedalus.novatech-llc.com
 
@@ -6,6 +8,10 @@ daedalus.novatech-llc.com
 
 [docker-hosts]
 daedalus.novatech-llc.com
+builder.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
+
+[builder]
+builder.novatech-llc.com
diff --git a/ansible-playbook/staging b/ansible-playbook/staging
index ad0ef4ac..8058586d 100644
--- a/ansible-playbook/staging
+++ b/ansible-playbook/staging
@@ -1,3 +1,5 @@
+builder.novatech-llc.com ansible_host=127.0.0.1 ansible_port=20022 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/bbworker/virtualbox/private_key'
+
 [bacula]
 testdaedalus.novatech-llc.com
 
@@ -6,6 +8,10 @@ testdaedalus.novatech-llc.com
 
 [docker-hosts]
 testdaedalus.novatech-llc.com
+builder.novatech-llc.com
 
 [buildsystem]
 buildsystem.novatech-llc.com
+
+[builder]
+builder.novatech-llc.com
